components/dashboard/src/admin/TeamsSearch.tsx

```diff
@@ -98,7 +98,7 @@ export function TeamsSearch() {
                             placeholder="Search Teams"
                             onKeyDown={(k) => k.key === "Enter" && search()}
                             onChange={(v) => {
-                                setSearchTerm(v.target.value.trim());
+                                setSearchTerm(v.target.value.trim().replace(/-/g, " "));
```

Comment:
But, Now this case will start failing 😞 

![image](https://user-images.githubusercontent.com/55068936/193438528-2cda50a2-cf95-470d-ac5d-43ef2bf0e78b.png)